### PR TITLE
Interactive branch

### PR DESCRIPTION
**Pull request by Metsanpeitto**
What I have implemented in this branch: 
- Add 2 new Js files and imported them as modules:
   - drag_drop.js
   - status.js
- Add event listeners to list items.
- Update the tasks object when a change occurs.
- Store the tasks object in local storage and retrieve it when needed.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello @Metsanpeitto 

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._

Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js [hunk 1]
```diff
@@ -2,9 +2,29 @@ import './style.css';
 import RecycleImg from './recycle.svg';
 import MoreImg from './more.svg';
 import TrashImg from './delete.svg';
+import { drag, drop, allowDrop } from './drag_drop.js';
+import updateTasks from './status.js';
```
Comment (mikenath223) [inline src/index.js:6]: - [ ] Loading any of the index.html shows a blank empty page. Why is this so? In this branch, you're expected to add interactiveness to the page not remove the previous work done. Kindly include the previous work done and ensure interactiveness is added to the todo list items.

Comment (mikenath223) [reply src/index.js:6]: - [ ] In addition, kindly ensure you only have a single `index.html` page.

Comment (Metsanpeitto) [reply src/index.js:6]: Hi Mike. 
Thanks for pointing out those problems. I have already solved them.
To run the app now you have different choices. 

### Setup

```cmd
git clone  https://github.com/Metsanpeitto/To-do-list_m2_w2.git
cd ./To-do-list_m2_w2
```

### Install

```cmd
npm install
```

### Usage

```cmd
npm start
```


or 

### Open HTML file

```cmd
open the ./dist/index.html in your favorite browser 
```

**Review by smcommits — APPROVED** [review]
### Status: Approved 👍🏽 

Hi @Metsanpeitto, 

## You have done a great job! Your project is complete. This PR is now approved. You can go ahead and merge the branch. Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**